tests/integration/test_pr_agent_workflow_specific.py

```
@@ -0,0 +1,492 @@
+                
+                assert version_count == 1, \
+                    f"Setup Python at line {i+1} has {version_count} python-version definitions, expected 1"
+
+
+class TestPRAgentWorkflowStructureValidation:
+    """Validate the overall structure of pr-agent.yml."""
+    
+    @pytest.fixture
+    def workflow_content(self) -> Dict[str, Any]:
+        """Load workflow content."""
+        with open('.github/workflows/pr-agent.yml', 'r', encoding='utf-8') as f:
```

Ruff (0.14.5)

Unnecessary mode argument

Remove mode argument

(UP015)